AccountData/build.gradle

```diff
@@ -23,28 +23,31 @@ android {
 }
 
 dependencies {
-  testCompile libs.junit
-  testCompile libs.assertjCore
-  testCompile libs.mockitoCore
-  testCompile libs.robolectric
-  testCompile "com.squareup.okhttp3:mockwebserver:$okHttpVersion"
-
-  compile libs.supportAnnotations
-  compile libs.rxjava
-  compile libs.retrofitConverterGson
-  compile libs.retrofitAdapterRxJava
-  compile libs.retrofit
-  compile libs.gson
-  provided project(":AccountDomain")
-  provided project(":TripKitDomain")
-
-  compile libs.dagger
+  testImplementation libs.junit
+  testImplementation libs.assertjCore
+  testImplementation libs.mockitoCore
+  testImplementation libs.robolectric
+  testImplementation "com.squareup.okhttp3:mockwebserver:$okHttpVersion"
+
+  implementation libs.supportAnnotations
+  implementation libs.rxjava
+  implementation libs.retrofitConverterGson
+  implementation libs.retrofitAdapterRxJava
+  implementation libs.retrofit
+  implementation libs.gson
+  compileOnly project(":AccountDomain")
+  compileOnly project(":TripKitDomain")
+
+  implementation libs.dagger
   kapt libs.daggerCompiler
-  kapt libs.value
-  provided libs.valueAnnotations
-  provided libs.builderAnnotations
-  provided libs.gsonAnnotations
-  compile libs.kotlin
+
+  annotationProcessor libs.value
```

Comment:
Should it be `kapt libs.value` instead?

Reply:
yes and no. 
yes to make the annotation processor to run because we use `kotlin-kapt` plugin so `annotationProcessor libs.value` is ignored.
No because we still need `annotationProcessor libs.value` to workaround an issue in the new android plugin. `libs.value` contains a processor and is added to the `implementation` classpath. In the new android plugin, annotation processors have to be declared explicitly. 
> If the plugin detects annotation processors on the compile classpath, your build fails and you get an error message that lists each annotation processor on the compile classpath. To fix the error, simply change the configuration of those dependencies to use annotationProcessor. If a dependency includes components that also need to be on the compile classpath, declare that dependency a second time and use the compile dependency configuration.